python/dask_cudf/dask_cudf/sorting.py

```diff
@@ -320,14 +320,14 @@ def get_default_shuffle_method():
     return default
 
 
-def _get_shuffle_type(shuffle):
-    # Utility to set the shuffle-kwarg default
+def _get_shuffle_type(shuffle_method):
```

Comment:
(Not required for this PR) Maybe this should be renamed to `_get_shuffle_method` to be explicit about it's purpose?

Reply:
Done in https://github.com/rapidsai/cudf/pull/14708/commits/cdb298b80f89ace961a3716da779774bb2d9a34e .